en/docs/consuming-services/create-an-application.md

```diff
@@ -1,51 +1,64 @@
-An application in Choreo is a logical representation of a physical application such as a mobile app, web app, device, etc. To consume an API in Choreo, you need to create an application that maps to your physical application and then subscribes to the required API over a usage policy plan that gives you a usage quota. A single application can have multiple subscriptions to APIs. Using the consumer key and consumer secret, you can generate an access token that you can use to invoke all the APIs subscribed to the same application.
+# Create an application in Choreo
```

Comment:
This does not require a title because it is reusable content used in both **Manage Applications** and **Consume a Service** pages. The content is also structured to allow reusability.